src/Raven.Server/Web/System/AdminTcpConnectionDebugInfoHandler.cs

```diff
@@ -0,0 +1,123 @@
+        public Task Statistics()
+        {
+            var properties = IPGlobalProperties.GetIPGlobalProperties();
+            var ipv4Stats = GetTcpStatisticsSafely(() => properties.GetTcpIPv4Statistics());
+            var ipv6Stats = GetTcpStatisticsSafely(() => properties.GetTcpIPv6Statistics());
+
+            var djv = new DynamicJsonValue
+            {
+                ["IPv4"] = ToDynamic(ipv4Stats),
+                ["IPv6"] = ToDynamic(ipv6Stats)
+            };
+
+            using (ServerStore.ContextPool.AllocateOperationContext(out JsonOperationContext context))
+            using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
+            {
+                context.Write(writer, djv);
+            }
+
+            return Task.CompletedTask;
+
+            static DynamicJsonValue ToDynamic(TcpStatistics stats)
+            {
+                var result = new DynamicJsonValue();
+                if (stats == null)
+                    return result;
+
+                result[nameof(stats.ConnectionsAccepted)] = stats.ConnectionsAccepted;
+                result[nameof(stats.ConnectionsInitiated)] = stats.ConnectionsInitiated;
+                result[nameof(stats.CumulativeConnections)] = stats.CumulativeConnections;
+                result[nameof(stats.CurrentConnections)] = stats.CurrentConnections;
+                result[nameof(stats.ErrorsReceived)] = stats.ErrorsReceived;
+                result[nameof(stats.FailedConnectionAttempts)] = stats.FailedConnectionAttempts;
+                result[nameof(stats.MaximumConnections)] = stats.MaximumConnections;
+                result[nameof(stats.MaximumTransmissionTimeout)] = stats.MaximumTransmissionTimeout;
+                result[nameof(stats.MinimumTransmissionTimeout)] = stats.MinimumTransmissionTimeout;
+                result[nameof(stats.ResetConnections)] = stats.ResetConnections;
+                result[nameof(stats.ResetsSent)] = stats.ResetsSent;
+                result[nameof(stats.SegmentsReceived)] = stats.SegmentsReceived;
+                result[nameof(stats.SegmentsResent)] = stats.SegmentsResent;
+                result[nameof(stats.SegmentsSent)] = stats.SegmentsSent;
+
+                return result;
+            }
+
+            static TcpStatistics GetTcpStatisticsSafely(Func<TcpStatistics> func)
+            {
+                try
+                {
+                    return func();
+                }
+                catch (PlatformNotSupportedException)
+                {
+                    return null;
+                }
+            }
+        }
+
+        [RavenAction("/admin/debug/info/tcp/active-connections", "GET", AuthorizationStatus.Operator, IsDebugInformationEndpoint = true)]
+        public Task ActiveConnections()
+        {
+            var properties = IPGlobalProperties.GetIPGlobalProperties();
+            var connections = properties.GetActiveTcpConnections();
```

Comment:
This can throw, no?

Reply:
No info in docs, but I got ex on a property where there was also no info, so I've added helper method to retrieve this safely.